docs/dev_guide/pr_review_procedure.rst

```diff
@@ -0,0 +1,81 @@
+This document describes the procedure that the people in the
+sunpy-maintainters group must follow when merging a pull request. If in
+any doubt about merging a pull request, the correct course of action is
+to contact the Lead Developer / Executive Director and ask.
+
+Fundamental Components of a Pull Request
+========================================
+
+Each pull request *must* meet the following criteria before it is
+considered for merge:
+
+-  The code must be PEP 8 compliant and meet the ill-defined SunPy
+   quality standards.
+-  The PR must contain a CHANGELOG entry if a sunpy-maintainter deems so.
+-  The test coverage **ABSOLUTELY** must not decrease, and for new
+   features should be at or very close to 100%.
+-  All code must be properly documented. Each function and each class
+   must have an associated doc string in the correct format.
+
+Review Process
+==============
+
+Before the ‘merge’ button is clicked the following criteria must be met:
+
+-  All the builds pass (currently Travis CI, Appveyor and Coveralls.
+-  At least two members (not the author of the PR) of the
+   sunpy-developers group have approved the PR. (Normally via the GitHub
+   `review`_ or by commenting on the PR with either a :+1:(``:+1:``)
+   emoticon or +1).
+-  All comments posted on the thread must be resolved, a +1 after any
+   comments by the same author indicates that the comments have been
+   satisfactorily resolved. (Note: do not +1 a pull request unless you
+   would be willing for it to be merged in it *current* state.)
```

Comment:
And we will avoid that extra *Note*. I may give a 👍 to some comment for its insight, and not for the PR.